source/ShipInfoPanel.cpp

```diff
@@ -644,9 +649,9 @@ void ShipInfoPanel::Rename(const string &name)
 
 
 
-bool ShipInfoPanel::CanDump() const
+bool ShipInfoPanel::CanDump()
 {
-	if(canEdit || shipIt == player.Ships().end())
+	if(panelState.CanEdit() || shipIt == panelState.Ships().end())
```

Comment:
Please consider to change panelState.Ships() to const or to provide an additional const version of the function.
If panelState.Ships() is also available in const version, then CanDump() can also stay const.